src/workers/cdc/primary.ts

```diff
@@ -710,6 +711,22 @@ const onSourceChange = async (
         flags: after,
       });
     }
+
+    const squadAfter = data.payload.after as ChangeObject<SquadSource>;
+    const squadBefore = data.payload.before as ChangeObject<SquadSource>;
+
+    if (
+      squadBefore.moderationRequired !== squadAfter.moderationRequired &&
+      !squadAfter.moderationRequired
+    ) {
+      await con.getRepository(SourcePostModeration).update(
+        {
+          sourceId: squadAfter.id,
+          status: SourcePostModerationStatus.Pending,
+        },
+        { status: SourcePostModerationStatus.Approved },
+      );
```

Comment:
Should we add a specific moderationReason so that we know when posts get approved this way, just so we can understand how much it happens?

Reply:
Not sure where we would use it but maybe we can check with product or @idoshamun

Reply:
I think we should have it just for our sake, so we can have insight on how the post was approved.

Reply:
also because this will not have `moderatedById` so we might think its some kind of issue on our side

Reply:
Yeah, do you have any suggestions on what should be the key to store in the `reason` column?

Reply:
whatever, some constant value like `moderationRequiredChanged` should do

Reply:
Cool. Let's do that.

Reply:
We should maybe change the column name since it is `rejectionReason` which is not applicable to this case anymore.

Reply:
ah its rejection reason, thought it was moderationReason 🤦 ... let's ignore it then it does not fit the schema

Reply:
We can add it to the `moderatorMessage` if you like.